lib/rules/symbol-description.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @fileoverview Rule to disallow or enforce description with the `Symbol`
+ * object
+ * @author Jarek Rencz
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+let astUtils = require("../ast-utils");
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+let MODE_ALWAYS = "always";
+let MODE_NEVER = "never";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "disallow or enforce description with the `Symbol`" +
+            " object",
+            category: "ECMAScript 6",
+            recommended: true
```

Comment:
This should be false (we can't update "ESLint: recommended" without a major release).